joomla/joomla-cms
# [4.0] Pre-update checks

### Summary of Changes

- Improve accessibility
- Improve language strings to match style guide
- Remove styling from language strings
- Revert removal of extension checks see #26177

### Testing Instructions
The pre-update check tab of com_joomlaupdate is only ever displayed when an update is available. To simulate this set the Update source to **custom** and the url to https://www.jah-tz.de/downloads/core/nightlies/next_major_list.xml

### Note
~still waiting an answer on #26177 but I have assumed that the removal was a mistake~
